.travis.yml

```diff
@@ -1,42 +1,14 @@
-sudo: false
-dist: precise
+#
+# Important: This file is for the wp-dev-lib Travis CI runs.
+# See sample-config/.travis.yml for a sample config for a WordPress project.
+#
 
-notifications:
-  email:
-    on_success: never
-    on_failure: change
-
-cache:
-  directories:
-    - node_modules
-    - vendor
-    - $HOME/phpunit-bin
-
-language:
-  - php
-  - node_js
-
-php:
-  - 5.2
-  - 5.3
-  - 5.6
-  - 7.0
+language: php
 
-env:
-  - WP_VERSION=latest WP_MULTISITE=0
-  - WP_VERSION=latest WP_MULTISITE=1
-  - WP_VERSION=trunk WP_MULTISITE=0
-  - WP_VERSION=trunk WP_MULTISITE=1
-
-install:
-  - nvm install 6 && nvm use 6
-  - export DEV_LIB_PATH=dev-lib
-  - if [ ! -e "$DEV_LIB_PATH" ] && [ -L .travis.yml ]; then export DEV_LIB_PATH=$( dirname $( readlink .travis.yml ) ); fi
-  - if [ ! -e "$DEV_LIB_PATH" ]; then git clone https://github.com/xwp/wp-dev-lib.git $DEV_LIB_PATH; fi
-  - source $DEV_LIB_PATH/travis.install.sh
+notifications:
+  email: false
 
 script:
-  - source $DEV_LIB_PATH/travis.script.sh
-
-after_script:
-  - source $DEV_LIB_PATH/travis.after_script.sh
+  - php -l *.php # Ensure valid PHP syntax
+  - shellcheck *.sh || true # Run shellcheck but ignore the output
+  - shellcheck --format json *.sh | grep --quiet --invert-match '"level":"error"' # Fail only if shellcheck finds errors.
```

Comment:
- Shellcheck returns the same exit code in case of warnings, notices and errors (see https://github.com/koalaman/shellcheck/issues/971).
- We make it produce JSON output and check if it contains `"level":"error"`.
- We invert the output of `grep` because we want it to pass if it _doesn't_ contain the string.